transcriptic/commands.py

```diff
@@ -1048,18 +1048,16 @@ def select_org(api, config, organization=None):
     click.echo(f"Logged in with organization: {organization}")
 
 
-def login(api, config, api_root=None, analytics=True, rsa_key=None):
+def login(api, config, api_root=None, analytics=True, rsa_key=None, web_staging=False):
     """Authenticate to your Transcriptic account."""
-    if api_root is None:
-        # Always default to the pre-defined api-root if possible, else use
-        # the secure.transcriptic.com domain
-        try:
-            api_root = api.api_root
-        except ValueError:
-            api_root = "https://secure.transcriptic.com"
-
+    # If user does not specify staging web api argument, default to the "https://secure.strateos.com" domain
+    if web_staging:
+        api_root = "https://webapp.staging.strateos.com/"
+    else:
+        api_root = "https://secure.strateos.com/"
```

Comment:
I think we may want to keep the secure.transcriptic.com domain here, since we haven't completely switched over yet and there are clients that are using the client to create Runs. I would also try to keep some of the error raising here.

Reply:
seconded.

additionally, we want to maintain the existing behavior for handling `api_root` if it is not None

Reply:
@joshhacksthings I'm not in favor of the existing behavior. Since it always looks for the api_root url from the .transcriptic file when api_root is none, you can't toggle back and forth between staging and production without having to manually change the .transcriptic file. Also, writing a separate staging login function instead of using a flag does not solve the issue because the .transcriptic config file gets modified either way. Are there any use cases where tx login defaulting to "https://secure.transcriptic.com" would cause errors for people?